documentation/jetty/README.adoc

```diff
@@ -0,0 +1,17 @@
+= Jetty Documentation
```

Comment:
We have a license header rule fo r`*.adoc` files.

Either add the license header file here, or you can exclude it in the top-level POM, around line 1928.

I prefer the latter.

Reply:
I'll just add it here since we are adding it for all other .adoc files. I just forgot about this one.